wire-ios-data-model/Source/Model/AccountStore.swift

```diff
@@ -94,18 +98,11 @@ public final class AccountStore: NSObject {
             try FileManager.default.removeItem(at: root.appendingPathComponent(directoryName))
             return true
         } catch {
-            log.error("Unable to remove all accounts at \(root): \(error)")
```

Comment:
`root` is a URL. URLs give identifying info about a user so don't think it should be logged. We have a ticket to change how logging works where we can make this safer: https://wearezeta.atlassian.net/browse/WPB-10227